locale/en.po

```diff
@@ -1042,21 +1046,46 @@ msgstr "run status:"
 msgid "SELECT_CELL_TYPE_FROM_EDITOR"
 msgstr "Select a cell type to edit here from the editor tab"
 
-#: ../src/engine/FPSCounter.cs:18 ../src/engine/PerformanceMetrics.cs:62
-#: ../src/engine/PerformanceMetrics.tscn:57
-msgid "FPS"
-msgstr "FPS: {0}"
-
 #: ../src/engine/LoadingScreen.cs:68 ../src/engine/LoadingScreen.tscn:171
 msgid "LOADING"
 msgstr "Loading"
 
-#: ../src/engine/PerformanceMetrics.cs:63
+#: ../src/engine/debugging/DebugOverlay.tscn:36
+msgid "DEBUG_PANEL"
+msgstr "Debug Panel"
+
+#: ../src/engine/debugging/DebugOverlay.tscn:46
+msgid "FPS_DISPLAY"
+msgstr "FPS display"
+
+#: ../src/engine/debugging/DebugOverlay.tscn:53
+msgid "PERFORMANCE_METRICS"
+msgstr "Performance Metrics"
+
+#: ../src/engine/debugging/DebugOverlay.tscn:60
+msgid "COLLISION_SHAPE"
+msgstr "Collision shapes"
```

Comment:
As it turn out that the collision shapes only show up after entity is spawned, I think it makes more sense to reword this to  "Spawn entity with collision shapes".